src/Query/ADQL_parser/ADQL_parser/init_numeric_geometry_function.cxx

```diff
@@ -38,15 +55,16 @@ void ADQL_parser::init_geometry ()
 
   column_or_number %= column_reference | boost::spirit::qi::double_;
   column_or_number.name ("column or number");
+
   coord %= column_or_number >> ',' > column_or_number;
   coord.name ("coordinate");
+
   point %= ascii::no_case["POINT"] >> '(' >> -(coord_sys > ',') > coord > ')';
   point.name ("point");
```

Comment:
`coord_sys` and `point` are also initialized in `init_point_or_column`. Should the initializations in this file be removed?

Reply:
Hmm.  query_server doesn't want me to rename libadql's 'geometry' to 'predicate_geometry_function'.  I'll try to add 'geometry' back as an alias to ease the transition.